ydlidar_ros2_driver_node:
  ros__parameters:
    # Configuração da porta serial
    port: /dev/ttyUSB0
    frame_id: laser_frame
    ignore_array: ""
    
    # Configurações de comunicação
    baudrate: 230400
    lidar_type: 1
    device_type: 0
    sample_rate: 9
    abnormal_check_count: 4
    
    # Configurações do scan
    resolution_fixed: true
    reversion: true
    inverted: true
    auto_reconnect: true
    isSingleChannel: false
    intensity: false
    support_motor_dtr: false
    
    # Limites angulares e de distância
    angle_max: 180.0
    angle_min: -180.0
    range_max: 12.0
    range_min: 0.15
    frequency: 10.0
    invalid_range_is_inf: false
